current-events/activity-current-events.md

```diff
@@ -0,0 +1,160 @@
+# Activity: Current Events
+
+## Goal
+
+Our goal is to create intentional space to discuss current events in technology.
+
+We want to become part of the conversation between technologists that evolves every day.
+
+Through discussion, we should aim to gain awareness of different topics and share what current events are interesting to us.
+
+Everyone will prepare a presentation of three or less slides to introduce a current event topic of your choice.
+
+## Preparation
+
+A slide deck of three or fewer slides should be prepared before activity day. Slide deck requirements are below.
+
+On the day of presentations, we should be prepared to discuss, present, and share in small groups.
+
+Open discussion about current events can affect us in many ways. Everyone should be prepared to engage in discussions however is best for you. This includes disengaging!
+
+## Activity Instructions
+
+1. Find an article/piece of media that describes a current event about technology that interests you.
+
+Current events should be topics actively relevant within the last twelve months. Some examples include:
+
+- A recent news article describing how tech is transforming
+- A documentary on some technology that was built in the last year
+- A thinkpiece on technology written recently
+- Instance of action, resistance, and/or justice related to technology and current events
+
+Avoid fiction, unless it is unambiguously making a direct connection to a current event.
+
+2. Create a slide deck of three or less slides describing the current event.
```

Comment:
countability
```suggestion
2. Create a slide deck of three or fewer slides describing the current event.
```

Reply:
I'd like it to be on record that I caught a less vs. fewer error of mine in..... some lesson. Hahaha. Thank you again!!

Reply:
I think it might have been in _this_ lesson! 😆